tests/integration/test_sharding.py

```diff
@@ -135,7 +146,7 @@ def test_shard_group_validation(cluster):
             '12345678901234567890123456789012',
             '1', '1',
             '1001', '20000', '1',
-            '1234567890123456789012345678901234567890', '1.1.1.1:1111') == b'OK'
+            '1234567890123456789012345678901234567890', '1.1.1.1:1111')
```

Comment:
do we want to `assert` ?

Reply:
This command will throw an exception. It's inside `with raises()` statement which is waiting to catch exception. So, assert() is there, just hidden.